jdaviz/app.vue

```diff
@@ -204,7 +204,7 @@ div.output_wrapper {
 }
 
 .v-toolbar__items .v-btn {
-  /* allow voolbar-items styling to pass through tooltip wrapping span */
+  /* allow toolbar-items styling to pass through tooltip wrapping span */
```

Comment:
Yes, I think it is possible indeed.

```suggestion
  /* allow v-toolbar-items styling to pass through tooltip wrapping span */
```